opencti-platform/opencti-graphql/src/modules/publicDashboard/publicDashboard-domain.ts

```diff
@@ -151,15 +166,23 @@ export const addPublicDashboard = async (
     [{ id: user.id, access_right: 'admin' }, { id: 'ALL', access_right: 'view' }],
     user,
   );
+
+  // check platform data sharing max markings
+  const availableDataSharingMarkingIds = (await getAvailableDataSharingMarkings(context, SYSTEM_USER)).map((m) => m.id);
+  if (input.allowed_markings_ids?.some((id) => !availableDataSharingMarkingIds.includes(id))) {
```

Comment:
there is no check of user allowed markings. I could as a user share a public dashboard with a marking > that the ones I have access to (via the API)

Reply:
done